includes/Dashboard.php

```diff
@@ -220,6 +221,31 @@ public function load_stories_dashboard() {
 		);
 	}
 
+	/**
+	 * Find status of site kit plugin in site.
+	 *
+	 * @since 1.1.0
+	 *
+	 * @return string|false if plugin is present return 'active' or 'inactive', otherwise return false
+	 */
+	public function determine_site_kit_plugin_status() {
+
+		// Get all plugins
+		$all_plugins = get_plugins();
+
+		// check if site kit is an installed plugin
+		if ( array_key_exists( 'google-site-kit/google-site-kit.php', $all_plugins ) ) {
```

Comment:
I'd do something like this:

```php
$is_site_kit_installed  = array_key_exists( 'google-site-kit/google-site-kit.php', $all_plugins )
$is_site_kit_active      = defined( 'GOOGLESITEKIT_VERSION' );
$analytics_module_active = false; // TODO: copy the logic from Analytics.php we need to share somehow.
$can_install_plugins     = current_user_can( 'install_plugins' )
$can_activate_plugin     = current_user_can( 'activate_plugin', 'google-site-kit/google-site-kit.php' );
```

Reply:
Unfortunately the `google-site-kit/google-site-kit.php'` for the _installed_ part is best we can do in WordPress :-(

It means we cannot detect whether Site Kit is installed in another directory (unlikely, but possible)

Reply:
thank you for the help here! I swapped out the structure and like that it's back to being all booleans.